packages/flutter/test/widgets/nested_scroll_view_test.dart

```diff
@@ -2424,7 +2424,8 @@ void main() {
       expect(tester.getCenter(find.text('Item 49')).dy, equals(585.0));
 
       // Overscroll, dragging like this will release with 0 velocity.
-      await tester.drag(find.text('Item 49'), const Offset(0.0, -50.0));
+      // , warnIfMissed: false
+      await tester.drag(find.text('Item 49'), const Offset(0.0, -50.0), warnIfMissed: false);
```

Comment:
Are these tests broken?

Reply:
I am working on these, even if the drag "misses", the scrollable is dragged as expected which is validated by the `expect` below. I added the `warnIfMissed` early in the PR, so it may not be necessary now.